test/README.md

```diff
@@ -0,0 +1,40 @@
+Full-Stack tests may be written for any given bug, they live in the test directory`.
+
+A test must be named using the issueXXXX-thing-fails-for-thing convention. A bash script ( with executable flag set ) and folder should be created with this name.
```

Comment:
Actually there's no real convention. And the bash script is not mandatory if it's just a package build. We should probably refactor this...